Makefile

```
@@ -54,22 +54,16 @@ endif
 govet:
 ifneq ($(HAS_SERVER),)
 	@echo Running govet
-	$(GO) get golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
-	$(GO) vet $$(go list ./server/...)
-	$(GO) vet -vettool=$(GOPATH)/bin/shadow $$(go list ./server/...)
+	@# Workaroung because you can't install binaries without adding them to go.mod 
+	cd /tmp	&& $(GO) get golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow@latest
```

```suggestion
	env GO111MODULE=off $(GO) get golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow@latest
```

Maybe better? Seems like this old functionality is an open issue.

---

Good idea :+1: